src/OpenTelemetry/Trace/Export/Internal/MultiProcessor.cs

```diff
@@ -0,0 +1,60 @@
+﻿// <copyright file="MultiProcessor.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace OpenTelemetry.Trace.Export
+{
+    internal class MultiProcessor : SpanProcessor
+    {
+        private readonly IEnumerable<SpanProcessor> processors;
+
+        public MultiProcessor(IEnumerable<SpanProcessor> processors)
+        {
+            this.processors = processors ?? throw new ArgumentNullException(nameof(processors));
```

Comment:
why not simply initialize this one without any child processors? Semantically it's the same as zero length of a colleciton

Reply:
just to be clear what is intention of this processor. If you want empty collection - pass empty collection. It's an internal API and we should know exactly how to use it, we should not leave room for different behavior.

Reply:
I'd say than we shouldn't allow empty collections =)

Reply:
sure, will do that